File: Javascript/simple stairs.js
function drawStairs(n) {
 return Array.from({length: n}, (_, i) => ' '.repeat(i) + 'I').join('\n')
}

const { assert } = require("chai")

describe("Tests", () => {
  it("test", () => {
    assert.strictEqual(drawStairs(1), "I");
    assert.strictEqual(drawStairs(3), "I\n I\n  I");
    assert.strictEqual(drawStairs(5), "I\n I\n  I\n   I\n    I");
  });
});

⬇️⬇️

/* Exampl(7)e：
I
 I
  I
   I
    I
     I
      I
*/

File: python/simple stairs.py
# https://www.codewars.com/kata/5b4e779c578c6a898e0005c5

def draw_stairs(n):
    return '\n'.join(' ' * i + 'I' for i in range(n))

stairs = '''I\n I\n  I'''
test.assert_equals(draw_stairs(3), stairs)

⬇️⬇️

"""Example (n = 7)：
I
 I
  I
   I
    I
     I
      I
"""
